pkg/util/mount/mount_windows.go

```diff
@@ -212,7 +212,7 @@ func getDeviceNameFromMount(mounter Interface, mountPath, pluginDir string) (str
 	if len(refs) == 0 {
 		return "", fmt.Errorf("directory %s is not mounted", mountPath)
 	}
-	basemountPath := normalizeWindowsPath(path.Join(pluginDir, MountsInGlobalPDPath))
```

Comment:
yes, callers within K8s have to append `MountsInGlobalPDPath` before making this call now. Picture this code living at `k8s.io/utils/mount` instead of within `k8s.io/kubernetes`. It can't/won't know anything about `MountsInGlobalPDPath`.

I made sure to update all callers.